po/common/common.pot

```diff
@@ -5385,12 +5385,12 @@ msgid "User is being deleted. This may take a few minutes."
 msgstr "User is being deleted. This may take a few minutes."
 
 msgctxt "attribute_environmental_score_not_applicable_title"
-msgid "Green-Score not yet applicable"
-msgstr "Green-Score not yet applicable"
+msgid "Green-Score not applicable"
+msgstr "Green-Score not applicable"
 
 msgctxt "attribute_environmental_score_not_applicable_description_short"
-msgid "Not yet applicable for the category"
-msgstr "Not yet applicable for the category"
+msgid "Not applicable for the category"
+msgstr "Not applicable for the category"
```

Comment:
Are we still using it then ?

Reply:
yes, this just changes the message for some categories like sodas and water. We might get a Green-Score for them at some point, but we don't necessarily want people to think it will come soon.